be/dtm0_log.c

```diff
@@ -101,6 +102,8 @@ M0_INTERNAL int m0_be_dtm0_log_init(struct m0_be_dtm0_log  *log,
 	m0_mutex_init(&log->dl_lock);
 	log->dl_is_persistent = is_plog;
 	log->dl_cs = cs;
+	if (is_plog)
+		log->dl_seg = seg;
```

Comment:
do we need:
```
else
        log->dl_seg = NULL;
```
I know this 'log' might have been set to all zero, but this will make sure the ->dl_seg is initialized.

Reply:
Good idea, fixed in https://github.com/Seagate/cortx-motr/pull/802/commits/d3efcfa00717f443705fe3bf9a6e3d27f5d63085
Now it is either a valid segment or NULL.